test/libsolidity/solidityExecutionFramework.h

```diff
@@ -236,6 +239,17 @@ class ExecutionFramework
 	};
 
 private:
+	struct sealEngineInit
```

Comment:
Same as for Mix, we don't need a struct for this.    It should just be done directly in a constructor, or in the main().